gpgwrapper/interfaces.go

```diff
@@ -21,10 +21,20 @@ import (
 	"github.com/fluidkeys/fluidkeys/fingerprint"
 )
 
+// ExportPrivateKeyInterface is the interface that wraps the ExportPrivateKey
+// method.
+//
+// ExportPrivateKey returns 1 ascii armored private key for the given
```

Comment:
This `ExportPrivateKey` comment should be above the function (not above the struct)